src/globus_sdk/services/flows/client.py

```diff
@@ -940,6 +955,68 @@ def __init__(
     def default_scope_requirements(self) -> list[Scope]:
         return [Scope(self.scopes.user)]
 
+    def add_app_transfer_data_access_scope(
+        self, collection_ids: UUIDLike | t.Iterable[UUIDLike]
+    ) -> Self:
+        """
+        Add a dependent ``data_access`` scope for one or more given ``collection_ids``
+        to this client's ``GlobusApp``, under the Transfer ``all`` scope.
+        Useful for preventing ``ConsentRequired`` errors when starting or resuming runs
+        of flows that use Globus Connect Server mapped collection(s).
```

Comment:
Clunky, but more precise (we still have users get confused requesting these scopes on HA collections).

```suggestion
        of flows that use non-HA Globus Connect Server mapped collection(s).
```